src/buttons.hpp

```diff
@@ -6,10 +6,40 @@
 
 #include "commands.hpp"
 #include "constants.hpp"
+/*
+three Button
+============
+                                                      vol/next-prev inverted
+           admin-menu  idle        pause       play        play
+-------------------------------------------------------------------------
+pause      select                  play        pause       pause
+pauseLong  end adm     shortcut    shortcut    track#      track#
+up         next                                next        vol++
+upLong     next 10     shortcut    shortcut    vol++       next
+down       prev                                prev        vol--
+downLong   prev 10     shortcut    shortcut    vol--       prev
```

Comment:
Same here, downLong with inverted buttons is implemented as pref, *cont.*.

Reply:
Same answer